obj/marshalMapWildcard.go

```diff
@@ -47,13 +47,19 @@ func (mach *marshalMachineMapWildcard) Reset(slab *marshalSlab, rv reflect.Value
 		mach.keys[i].rv = v
 		mach.keys[i].s = v.String()
 	}
-	switch mach.cfg.MapMorphism.KeySortMode {
+
+	ksm := atlas.KeySortMode_Default
+	if mach.morphism != nil {
```

Comment:
I don't think this branch should ever be reachable.  The marshaller *always* sets it during dispatch.

Reply:
I don't think thats true... If i take this out i get panics in tests